### ISSUE TITLE
Scale usage numbers in overview page

### ISSUE THREAD

**Issue by nlcwong** [project-koku/koku-ui#935]
**Describe the bug**
In the overview page, the usage cards, when the cost and/or usage numbers become big (greater than 4 digits), the responsiveness of the card's layout needs to also include a way to scale the numbers to be more readable.  (e.g. "400,000 core-hours" could be "400k core-hours")

**To Reproduce**
1. Go to https://cloud.redhat.com/hybrid/cost-management
2. Click on Cloud tab
3. Scroll down to "Compute EC2" usage card
4. Make sure there are accounts that has usage/cost greater than 10,000.

**Expected behavior**
TBD - will need to consult with UX content strategy.
Suggestions: when the usage or cost goes beyond 4 digits, the UI should respond by either scaling the number or making the layout more responsive sooner or truncation?

**Screenshots**
![Cost-Overview-Cloud-NumbersWontScale](https://user-images.githubusercontent.com/31672086/61904630-34f1e780-aef5-11e9-8a64-89b232478781.png)

**Desktop (please complete the following information):**
All

**Smartphone (please complete the following information):**
All

**Additional context**
The account would need to have usage/cost that is beyond the thousands, usually the hundred thousands.

**Comment by nlcwong** [comment]
This issue was originally created in Jira, its been transposed to GH.
https://projects.engineering.redhat.com/browse/RHICOST-1

**Comment by boaz0** [comment]
we should bubble this up to the patternfly core team and in meanwhile override the css to fix that problem. 

//cc @dlabrecq

**Comment by dlabrecq** [comment]
This looks like a very small screen, judging by the width of the cards. That is, much smaller than a 15"  laptop.

We will likely need to change the layout to accommodate large numbers. For example, place cost and hours on their own line.

Not certain this is a PatternFly issue. There just isn't much room in the cards to have costs side-by-side. At least, not at this size screen.

**Comment by boaz0** [comment]
If the screen is the problem then why not using grid system and set a different layout on mobile/tablet devices?

**Comment by dlabrecq** [comment]
Yes, a grid would be used to "place cost and hours on their own line when the screen is less than 1200px."

**Comment by nlcwong** [comment]
@dlabrecq @boaz0 Even with the grid layout, would we still need a plan to figure out how to scale the numbers when the card reaches its minimum?  If so, how many digits do you think we could have until we need to scale?

**Comment by boaz0** [comment]
@nlcwong you're right. We can solve this by shrinking values like $3,340,212 into $ 3.34M and have a tooltip with the accurate value ($3,340,212).

**Comment by dlabrecq** [comment]
We can fit about 14 chars in a card (using the font size below), including commas, before we need to truncate. That on a screen about 1100px wide.

![Screen Shot 2019-07-29 at 1 40 47 PM](https://user-images.githubusercontent.com/17481322/62069564-93c1a480-b206-11e9-884c-bba8b991fb51.png)
